<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mosfet BYD</string>
  </property>
  <widget class="QGroupBox" name="groupBox_connect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>1361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,1,2,2,2">
    <property name="leftMargin">
     <number>18</number>
    </property>
    <property name="topMargin">
     <number>18</number>
    </property>
    <property name="rightMargin">
     <number>18</number>
    </property>
    <property name="bottomMargin">
     <number>18</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_hardware">
        <property name="text">
         <string>Hardware:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbBox_hardware">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>2</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>36</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushBtn_refresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
      <item>
       <widget class="QLabel" name="label_mode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Mode:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbBox_mode">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_buadrate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Buadrate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbBox_buadrate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushBtn_disconnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushBtn_connect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_Action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>1351</width>
     <height>92</height>
    </rect>
   </property>
   <property name="title">
    <string>Window Control</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioBtn_nocmd">
        <property name="text">
         <string>NoCMD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn_manualup">
        <property name="text">
         <string>Manual Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn_manualdown">
        <property name="text">
         <string>Manual Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn_autoup">
        <property name="text">
         <string>Auto Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn_autodown">
        <property name="text">
         <string>Auto Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
